### PR TITLE
Add tasks to the packer namespace to copy AMIs to other regions and tag them

### PR DESCRIPTION
**Pull request by jmanero-r7**
This PR adds three tasks to the packer namespace:
- `copy PROFILE` wraps calls to `EC2.copy_image` to each AWS region in the `ami_regions` list for each packer build in PROFILE
- `wait PROFILE` polls `describe_images` for each of the above images every 10 seconds until every image's state is `available`
- `tag PROFILE` calls `create_tags` on each of the above images with the tags from the respective source-image. It adds a `source_region` and `source_ami` tag to those already attached to the sourcve-image.

### CODE REVIEW

File: lib/builderator/tasks/packer.rb [hunk 1]
```diff
@@ -28,6 +32,59 @@ def build(profile = :default, *args)
...
+
+      desc 'copy PROFILE *REGIONS', 'Copy AMIs generated by packer to REGIONS'
+      def copy(profile, *regions)
+        invoke :configure, [profile], options
+
+        regions.each do |region|
+          images.each do |image_name, image|
+            say_status :copy, "AMI #{image_name} (#{image.image_id}) from #{Config.aws.region} to #{region}"
+
+            client(region).copy_image(:source_region => Config.aws.region,
```
Comment (athompson-r7) [inline lib/builderator/tasks/packer.rb]: I think we can use `ami_regions` in the packer.json to do this, given a recent version of packer.

https://www.packer.io/docs/builders/amazon-ebs.html#ami_regions

File: lib/builderator/tasks/packer.rb [hunk 2]
```diff
@@ -28,6 +32,110 @@ def build(profile = :default, *args)
...
+          build.ami_regions.each do |region|
+            regional_image = Util.ec2(region).describe_images(:filters => filters).images.first
+
+            say_status :tag, "AMI #{image_name} (#{regional_image.image_id}) in #{region}"
+            Util.ec2(region).create_tags(:resources => [regional_image.image_id], :tags => image.tags)
+          end
+        end
+      end
+
+      desc 'wait PROFILE', 'Wait for coppied AMIs to become available in another region'
```
Comment (dgreene-r7) [inline lib/builderator/tasks/packer.rb]: Spelling

File: lib/builderator/tasks/packer.rb [hunk 3]
```diff
@@ -28,6 +32,110 @@ def build(profile = :default, *args)
...
+
+              say_status :image, "#{image_id} (#{image.name}) is #{state} in #{region}", status_color
+            end
+          end
+
+          ## If waiting == false, loop immediately to break
+          sleep(10) if waiting
+        end
+
+        say_status :complete, 'All coppied images are available'
```
Comment (dgreene-r7) [inline lib/builderator/tasks/packer.rb]: Spelling

**Comment by dgreene-r7** [comment]
Other than spelling looks ok to me. You should probably have someone more familiar with the builderator codebase (@athompson-r7) take a look to confirm, though.

**Pull request merged**